test/linters_test.go

```diff
@@ -110,7 +110,7 @@ func testOneSource(t *testing.T, sourcePath string) {
 		"--print-issued-lines=false",
 		"--print-linter-name=false",
 		"--out-format=line-number",
-		"--max-same-issues=10",
+		"--max-same-issues=100",
```

Comment:
Why have you decided to increase the limit?

Reply:
Output all the results will enable to all test pass correctly.


When "--max-same-issues=10"

```sh
$ T=noctx.go make test_linters       
GL_TEST_RUN=1 go test -v ./test -count 1 -run TestSourcesFromTestdataWithIssuesDir/noctx.go
=== RUN   TestSourcesFromTestdataWithIssuesDir
=== RUN   TestSourcesFromTestdataWithIssuesDir/noctx.go
=== PAUSE TestSourcesFromTestdataWithIssuesDir/noctx.go
=== CONT  TestSourcesFromTestdataWithIssuesDir/noctx.go
--- FAIL: TestSourcesFromTestdataWithIssuesDir (1.28s)
    linters_test.go:41: testdata/*.go
    --- FAIL: TestSourcesFromTestdataWithIssuesDir/noctx.go (1.42s)
        linters_test.go:142: [run --allow-parallel-runners --disable-all --print-issued-lines=false --print-linter-name=false --out-format=line-number --max-same-issues=10 -Enoctx --no-config testdata/noctx.go]
        linters_test.go:37:
                Error Trace:    linters_test.go:37
                                                        linters_test.go:143
                                                        linters_test.go:57
                Error:          Received unexpected error:

                                noctx.go:34: missing error "should rewrite http.NewRequestWithContext or add \\(\\*Request\\).WithContext"
                                noctx.go:47: missing error "should rewrite http.NewRequestWithContext or add \\(\\*Request\\).WithContext"
                                noctx.go:69: missing error "should rewrite http.NewRequestWithContext or add \\(\\*Request\\).WithContext"
                                noctx.go:122: missing error "should rewrite http.NewRequestWithContext or add \\(\\*Request\\).WithContext"
                Test:           TestSourcesFromTestdataWithIssuesDir/noctx.go
FAIL
FAIL    github.com/golangci/golangci-lint/test  3.172s
FAIL
make: *** [Makefile:38: test_linters] Error 1
```